tox.ini

```diff
@@ -229,12 +196,33 @@ commands = {[with-cython]commands}
 setenv = {[with-cython]setenv}
 deps = {[with-cython]deps}
        gunicorn
-       # NOTE(caselit): meinheld requires greenlet<0.5 that's not compatible with py3.10
-       meinheld
        uwsgi
        waitress
 commands = pytest -v tests/test_wsgi_servers.py
 
+[testenv:wsgi_meinheld]
+# NOTE(caselit): meinheld requires greenlet<0.5 that's not compatible with py3.10
+basepython = python3.8
```

Comment:
could think about using 3.9 since 3.8 will be eol soon

Reply:
I'm not sure if Meinheld works with 3.9. I think those old benchmarks unfortunately don't, so I kept everything on 3.8 for now.

Reply:
(These tests will only be run on master merges, not PR.)